docs/cel2/l2-operator-guide.md

```diff
@@ -179,7 +179,6 @@ docker run -d \
     --rollup.config=/data/rollup.json \
     --verifier.l1-confs=4 \
     --rpc.addr=127.0.0.1 \
-    --syncmode=<consensus-layer/execution-layer>
```

Comment:
I think `--syncmode` is necessary here:

```
/ # op-node --help | grep -A1 syncmode
    --syncmode value                    (default: consensus-layer)         ($OP_NODE_SYNCMODE)
          Blockchain sync mode (options: consensus-layer, execution-layer)
```

Reply:
The default is to use consensus-layer, and the instructions before this block (line 161) say to append `--syncmode=execution-layer` if you want to use snap sync. That seems reasonable to me.